packages/cli/src/utils/installationInfo.ts

```diff
@@ -0,0 +1,174 @@
+/**
+ * @license
+ * Copyright 2025 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { isGitRepository } from '@google/gemini-cli-core';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as childProcess from 'child_process';
+
+export enum PackageManager {
```

Comment:
impressive list of supported package managers!

Reply:
Thanks!